raiden-dapp/tests/unit/components/channel-list.spec.ts

```diff
@@ -119,181 +107,134 @@ describe('ChannelList.vue', () => {
   });
 
   describe('close a channel', () => {
-    test('close the channel when the user confirms', async () => {
-      raiden.closeChannel = jest.fn().mockReturnValue(null);
+    beforeEach(() => {
+      raiden.closeChannel = jest.fn();
+    });
+
+    test('display confirmation dialog when close is pressed', async () => {
       wrapper.find('#channel-278').trigger('click');
-      await wrapper.vm.$nextTick();
```

Comment:
Is there any specific reason that the call to the function is removed here? `$nextTick()` and `$nextTick` are not the same. One is calling the function and the other is the function.

Reply:
No, that must be an error on my side, I will revert it.